client/src/components/CreatePost/Form/FirstSection.js

```diff
@@ -25,11 +36,12 @@ const First = ({
         onChange={onChangeTitle}
         value={formData.title}
         placeholder="Title"
+        maxLength={35}
```

Comment:
I don't think 35 chars is enough though. that can be like 3-5 words... We should double up this or at least 60 chars I would say

Reply:
What do you think @joshmorel ? isn't 35 chars too short???